service/src/main/kotlin/xyz/lebkuchenfm/Application.kt

```diff
@@ -117,13 +121,18 @@ fun Application.module() {
         }
     }
 
+    install(WebSockets) {
```

Comment:
I feel like we might want to move all those `install`s to separate method
- our own helper with all plugins
or
- move socket installation to eventStream route https://ktor.io/docs/server-plugins.html#install-route

Reply:
I feel that it's better the way it's now. It's not even 150 lines long, and I find it more readable that the whole configuration is in single file.